src/functional.test.ts

```diff
@@ -289,5 +272,115 @@ describe('create-release-branch (functional)', () => {
         },
       );
     });
+
+    it('errors before making any changes if the edited release spec omits changed packages', async () => {
+      await withMonorepoProjectEnvironment(
+        {
+          packages: {
+            $root$: {
+              name: '@scope/monorepo',
+              version: '20220101.1.0',
+              directoryPath: '.',
+            },
+            a: {
+              name: '@scope/a',
+              version: '1.0.0',
+              directoryPath: 'packages/a',
+            },
+            b: {
+              name: '@scope/b',
+              version: '1.0.0',
+              directoryPath: 'packages/b',
+            },
+            c: {
+              name: '@scope/c',
+              version: '1.0.0',
+              directoryPath: 'packages/c',
+            },
+            d: {
+              name: '@scope/d',
+              version: '1.0.0',
+              directoryPath: 'packages/d',
+            },
+          },
+          workspaces: {
+            '.': ['packages/*'],
+          },
+        },
+        async (environment) => {
+          await expect(
+            environment.runTool({
+              releaseSpecification: {
+                packages: {
+                  a: 'major',
+                  c: 'patch',
+                },
+              },
+            }),
+          ).toThrowExecaError(
+            `
+Error: Your release spec could not be processed due to the following issues:
+
+* The following packages, which have changed since their latest release, are missing.
+
+  - @scope/b
+  - @scope/d
+
+  Consider including them in the release spec so that any packages that rely on them won't break in production.
+
+  If you are ABSOLUTELY SURE that this won't occur, however, and want to postpone the release of a package, then list it with a directive of "intentionally-skip". For example:
+
+    packages:
+      "@scope/b": intentionally-skip
```

Comment:
Could you add a test to cover the `intentionally-skip` flag as well?

Reply:
Good catch. Added here: [6e94e30](https://github.com/MetaMask/create-release-branch/pull/29/commits/6e94e30f40cbe1d5f6106b500ff7d9d4f23834ea)